addons/editor-block-chomping/addon.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://raw.githubusercontent.com/ScratchAddons/manifest-schema/dist/schema.json",
+  "name": "Extend wrapped C blocks",
+  "description": "Makes C blocks extend when wrapping them around blocks, like in Scratch 2.0.",
+  "credits": [
+    {
+      "name": "CST1229",
+      "link": "https://github.com/CST1229/"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["projects"]
+    }
+  ],
+  "tags": ["editor", "codeEditor"],
```

Comment:
Maybe this could be considered a theme?

Reply:
I guess? As custom block shape is also a theme.